core/templates/pages/feedback-updates-page/feedback-updates-page.component.html

```diff
@@ -46,7 +46,7 @@
               <a class="btn oppia-dashboard-intro-button oppia-transition-200 oppia-feedback-updates-intro-button-link"
                  [innerHTML]="'I18N_ACTION_BROWSE_LESSONS' | translate"
                  href="/learn/math"
-                 [smartRouterLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.CLASSROOM.ROUTE">
```

Comment:
Instead of hardcoding the URL like so, is it possible to fix the variables involved?

Reply:
Done. PTAL @Lawful2002